test/system/070-build.bats

```diff
@@ -177,6 +177,20 @@ Labels.$label_name | $label_value
     run_podman rmi -f build_test
 }
 
+@test "podman build - stdin test" {
+    if is_remote && is_rootless; then
+        skip "unreliable with podman-remote and rootless; #2972"
+    fi
+
+    PODMAN_TIMEOUT=240 run_podman build -t build_test - << EOF
+FROM alpine
```

Comment:
Could you make this $IMAGE (instead of alpine), to avoid hitting a registry?

Reply:
If that works, sure.